index.css

```diff
@@ -0,0 +1,617 @@
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 32px;
+  color: #fff;
+}
+
+.page-title {
+  color: #ec5242;
+  font-size: 1.2rem;
+  letter-spacing: 2px;
+  line-height: 21px;
+  font-weight: 800;
+  padding-left: 10px;
+}
+
+.page-hello {
+  color: #ec5242;
+  padding-left: 10px;
+  margin-bottom: 0;
+  letter-spacing: 0;
+}
+
+.close {
+  color: #aaa;
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
+}
+
+.close:hover,
+.close:focus {
+  color: #000;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.icon {
+  font-size: 1rem;
+  color: #fff;
+}
+
+#top-panel {
+  background-color: #3e3e3e;
+  width: 100%;
+  display: none;
+}
+
+#nav-bar {
+  border: none;
+}
+
+.w-max {
+  max-width: 42px;
+  vertical-align: text-bottom;
+  border: 1px solid rgb(236, 236, 236);
+}
+
+.w-max-1 {
+  max-width: 100px;
+  border: 1px solid rgb(236, 236, 236);
+}
+
+.w-max-3 {
+  width: 100%;
+}
+
+.main-bg {
+  background-image: url('/assets/header-bg.png');
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  border: 1px solid rgba(39, 42, 50, 0.24);
+}
+
+/* .main-about-bg {
+  background-image: url('/assets/bg-sponsors.jpg');
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+} */
+
+.message-1,
+.message-2,
+.message-3,
+.message-4 {
+  text-align: center;
+  margin: 15px 0 25px 0;
+  line-height: 160%;
+  padding: 10px 16px;
+}
+
+.message-1 {
+  font-family: 'cocogoose', sans-serif;
+  border: 4px solid rgb(255, 255, 255);
+  background-color: rgb(247, 247, 248);
+  font-size: 11px;
+  font-weight: normal;
+  color: #404040;
+  line-height: 23px;
+}
+
+.message-2 {
+  font-family: 'cocogoose', sans-serif;
+  background-color: rgba(247, 247, 248, 0.65);
+  font-size: 10px;
+  font-weight: normal;
+  color: #333;
+}
+
+.message-3 {
+  font-family: 'poppins', sans-serif;
+  background-color: rgba(247, 247, 248, 0.65);
+  font-size: 12px;
+  font-weight: bold;
+  color: #333;
+  text-decoration: underline;
+}
+
+.message-4 {
+  font-family: 'cocogoose', sans-serif;
+  font-size: 10px;
+  font-weight: bold;
+  color: #7a7a7a;
+}
+
+.date-info {
+  font: bold 22px Verdana, sans-serif;
+  color: rgb(87, 87, 87);
+  letter-spacing: 0;
+  margin-bottom: 10px;
+  padding: 0 30px;
+}
+
+.locate {
+  font-family: 'cocogoose', sans-serif;
+  font-size: 0.6rem;
+  padding: 10px;
+  font-weight: bold;
+  color: #3e3e3e;
+}
+
+.program-bg {
+  background-repeat: repeat;
+  background-color: rgb(39, 42, 50);
+  background-image: url('/assets/bg-sponsors.jpg');
+}
+
+.program-section {
+  /* display: grid; */
+  grid-template-columns: 15% 30% 55%;
+  align-items: center;
+  justify-content: space-between;
+  background-color: rgba(255, 255, 255, 0.15);
+  transition: all 0.3s ease-in-out;
+}
+
+.program-section:hover {
+  border: 1px solid #fff;
+  cursor: pointer;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 0 20px;
+}
```

Comment:
- [ ] Great job with your design 👍 . Kindly layout the `main programs` mobile  home page as seen in the original design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/190472163-ac70e369-37f6-401f-812d-2aa1bc8c546c.png)  |  ![image](https://user-images.githubusercontent.com/74814780/190472040-76c50122-8bf3-4eb5-8838-f753003fc434.png)